BedrockServer.cpp

```diff
@@ -2006,6 +2021,11 @@ void BedrockServer::_acceptSockets() {
     // Try each port.
```

Comment:
After looking at the code, we also need to lock around the `for` loop above on `_portPluginMap`.

Reply:
Good call, updated